tests/sentry/snuba/metrics/test_query_builder.py

```diff
@@ -1015,6 +1044,87 @@ def test_translate_meta_results_with_duplicates():
     )
 
 
+@mock.patch(
+    "sentry.snuba.metrics.query_builder.get_metric_object_from_metric_field",
+    return_value=SingularEntityDerivedMetric(
```

Comment:
I prefer to extract this mocked metric to variable, to simplify usage of `mock.patch` decorator.

Reply:
feel like it is unnecessary to define a variable for this mocked variable that is used only once tbh

Reply:
correct, my point more about:
In ideal world you do this: patch and replace with mock object, then call you code. And important to check that mocked has been called also using assert_called_with instead of `_`.

It helps to prevent to some flakiness in future.

But i'm fine to keep as is.